modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java

```diff
@@ -171,8 +173,7 @@ private void createUser(String familyName, String givenName, String userName, St
         request.setEntity(entity);
 
         HttpResponse response = client.execute(request);
-        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " +
-                "has not been created successfully");
+        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " + "has not been created successfully");
```

Comment:
Shall we have "User" and the rest of the sentence as a single string

Reply:
Fixed